### ISSUE TITLE
ScaleBarWidget can not draw because the Map.CRS is not set

### ISSUE THREAD

**Issue by pauldendulk** [Mapsui/Mapsui#1218]
This is the error seen by our user:

```
2021-09-17 12:51:17.839863+0200 caminomobileapp.iOS[32980:1360459] MAPSUI Error: ScaleBarWidget can not draw because the Map.CRS is not set -
2021-09-17 12:51:18.411406+0200 caminomobileapp.iOS[32980:1360459] MAPSUI Error: ScaleBarWidget can not draw because 
```

Perhaps the widget should assume the SphericalMercator projection in this case.

**Comment by pauldendulk** [comment]
@dinonovak Could it be you do add the ScaleBarWidget somewhere in your own code? 

map.Widgets.Add(new ScaleBarWidget(....

**Comment by dinonovak** [comment]
@pauldendulk yes I was adding ScaleBarWidget in my code

**Comment by charlenni** [comment]
If you want to use ScaleBarWidget, then you have to initialize the map like this:

            var map = new Map
            {
                CRS = "EPSG:3857",
                Transformation = new MinimalTransformation()
            };

If you don't do it this way, the ScaleBarWidgt don't know, what type of CRS the map has and couldn't calculate any values.

**Comment by charlenni** [comment]
@pauldendulk This is only included in the Forms documentation. Perhaps you want to set this values as default? Or you have to mention this on all platforms.

**Comment by janusw** [comment]
> Perhaps you want to set this values as default? 

I agree that reasonable default values would be useful. 👍

**Comment by janusw** [comment]
This is fixed via #1983.
